Maquetado/application.css

```diff
@@ -0,0 +1,90 @@
+* {
```

Comment:
Modify your .gitignore file to avoid upload *.css and binaries files. Please delete these files.